docs/src/quickstart.md

```diff
@@ -1,4 +1,182 @@
 Quick Start Guide
 =================
 
-TODO: Quick example of solving an LP and getting the solution back.
+This quick start guide will introduce the main concepts of JuMP. If you are
+familiar with another modeling language embedded in a high-level language such
+as PuLP (Python) or a solver-specific interface you will find most of this
+familiar, with the exception of *macros*. A deep understanding of macros is not
+essential, but if you would like to know more please see the
```

Comment:
Reading this, it seems to me that it would be a good idea to first understanding macros while this really  not a good idea to learn metaprogramming to use JuMP. This might discourage many users. I don't know if essential is the weak enough, maybe "needed" or "required" ? Not sure :)